ocs_ci/ocs/bucket_utils.py

```diff
@@ -36,20 +36,28 @@ def craft_s3_command(cmd, mcg_obj=None, api=False, signed_request_creds=None):
     """
     api = "api" if api else ""
     if mcg_obj:
+        if mcg_obj.region:
+            region = f"AWS_DEFAULT_REGION={mcg_obj.region} "
```

Comment:
Is this AWS only, or would it work on Azure, GCP, OpenStack and other providers?

Reply:
This is done so that other cloud providers that don't use region during authentication (like RGW) don't need to specify it.

Reply:
It is indeed confusing but from what I see, this is for creating the s3 command on the AWS CLI pod which is used by all platforms

Reply:
Yes, aws cli is used for s3 utilization of all platforms. AWS_DEFAULT_REGION, AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY are aws cli specific variables but credentials contained in them should work for all s3 compatible interfaces.